app/views/shared/_changes_to_eligible_page.erb

```diff
@@ -0,0 +1,9 @@
+<br>
+<h2 class="govuk-heading-m">Content changes to the eligible page (eligibility checker)</h2>
+
+<p class="govuk-body govuk-!-font-weight-bold">Add additional information for sections:</p>
```

Comment:
This should probably be a heading `<h3 class="govuk-heading-s">` to show that it's a marker of a different section.